src/main/java/uk/gov/hmcts/reform/preapi/entities/User.java

```diff
@@ -35,6 +38,14 @@ public class User extends CreatedModifiedAtEntity {
     @Column(name = "deleted_at")
     private Timestamp deletedAt;
 
+    @OneToMany
+    @JoinColumn(name = "user_id", referencedColumnName = "id", nullable = true)
+    private Set<AppAccess> appAccess;
+
+    @OneToMany
+    @JoinColumn(name = "user_id", referencedColumnName = "id", nullable = true)
+    private Set<PortalAccess> portalAccess;
+
```

Comment:
should these not be 1-1?

Reply:
Going to leave this as discussed due to future requirements on app access